lib/cmd/common/build.js

```diff
@@ -127,16 +127,18 @@ function validateArgsNgui(args) {
 
 // convert our args..
 function argsToPayload(args) {
-  var initialPayload = "generateSrc=false";
-  return _.reduce(args, function(payload, value, key) {
-    if (_.isUndefined(value)) {
-      return payload;
-    }
-    if (_.isObject(value)) {
-      value = JSON.stringify(value);
-    }
-    return payload + "&" + key + "=" + value;
-  }, initialPayload);
+  args.generateSrc=false;
+  return args;
+  // var initialPayload = "generateSrc=false";
+  // return _.reduce(args, function(payload, value, key) {
+  //   if (_.isUndefined(value)) {
+  //     return payload;
+  //   }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)